src/wallet/wallet.h

```diff
@@ -1056,9 +1056,9 @@ class CWallet : public CCryptoKeyStore, public CValidationInterface
     CPubKey GenerateNewHDMasterKey();
     
     /* Set the current HD master key (will reset the chain child index counters)
-       If possibleOldChain is provided, the parameters from the old chain (version)
-       will be preserved. */
-    bool SetHDMasterKey(const CPubKey& key, CHDChain *possibleOldChain = nullptr);
+       Sets the master keys version based on the current wallet version (so make
+       its set appropriately before calling this). */
```

Comment:
Couple of grammar/typos here. Suggest you replace with: `The master key version is set to the current wallet version, so the caller must ensure that current wallet version is correct before calling this function.` or similar